packages/trading-widget/src/core-kit/hooks/trading/withdraw-v2/use-min-withdraw-slippage.ts

```diff
@@ -0,0 +1,49 @@
+import { useEffect } from 'react'
+
+import {
+  useIsDepositTradingPanelType,
+  useReceiveTokenInput,
+  useTradingPanelSettings,
+} from 'core-kit/hooks/state'
+import {
+  useIsWithdrawSwapStep,
+  useSwapReceiveValueDiff,
+} from 'core-kit/hooks/trading/withdraw-v2/swap-step'
+import { useConfigContextParams } from 'trading-widget/providers/config-provider'
+
+export const useMinWithdrawSlippage = () => {
+  const { defaultWithdrawSlippage, defaultSwapTransactionSlippage } =
+    useConfigContextParams()
+  const isWithdraw = !useIsDepositTradingPanelType()
+  const isWithdrawSwapStep = useIsWithdrawSwapStep()
```

Comment:
Would it be helpful  to return  also `isFetching`  state from `useIsWithdrawSwapStep`  to make effect more stable from flicks?

Reply:
I added prefetch of `trackedAssets` on deposit tab to prevent flicks, but will also add `isFetching` state to be on a safe side